doc/api/examples/codeRepositorySettings.json

```diff
@@ -0,0 +1,45 @@
+{
+    "projectUrl": "https://mantik.ai/projects/1",
+    "environment": "HPC",
+    "deploymentModes": [
+        {
+            "$ref": "deploymentMode1.json"
+        },
+        {
+            "$ref": "deploymentMode2.json"
```

Comment:
`deploymentMode2.json` refers to an AWS deployment, somehow conflicting with `environment: HPC` in line 3. Might be irellevant for the example, but please check again.

This actually spawns the question, whether a code repository should have multiple `environment` entries, or of this should be decoupled altogether.

Reply:
Rename `deploymentMode` to `trainingInfrastructure`.